core/src/main/scala-3/spire/syntax/Ops.scala

```diff
@@ -83,7 +83,7 @@ final class SignedOps[A](a: A)(using s: Signed[A]):
   def isSignNonNegative: Boolean = s.isSignNonNegative(a)
 
 final class TruncatedDivisionOps[A](lhs: A)(using ev: TruncatedDivision[A]):
-  def toBigIntOpt: Opt[BigInt] = ev.toBigIntOpt(lhs)
+  // def toBigIntOpt: Opt[BigInt] = ev.toBigIntOpt(lhs) // TODO port to algebra?
```

Comment:
This seems to be an API change, why?

Reply:
Could it be implemented in `spire`?

Reply:
For some reason it wasn't included in https://github.com/typelevel/algebra/pull/247.

I think it would be good if we could continue to implement it in spire, but I wasn't sure how unless we introduce a new typeclass? Any ideas?